crates/core/c8y_api/src/smartrest/alarm.rs

```diff
@@ -8,18 +8,17 @@ pub fn serialize_alarm(c8y_alarm: &C8yAlarm) -> Result<String, time::error::Form
     let smartrest = match c8y_alarm {
         C8yAlarm::Create(alarm) => {
             let smartrest_code = match alarm.severity {
-                AlarmSeverity::Critical => 301,
```

Comment:
As per usual, I bumped into more cases of SmartREST being generated in non-ideal ways, so I tidied that up too in a separate commit